pkgs/development/python-modules/langgraph-checkpoint/default.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  buildPythonPackage,
+  dataclasses-json,
+  fetchFromGitHub,
+  langchain-core,
+  langgraph-sdk,
+  poetry-core,
+  pytest-asyncio,
+  pytestCheckHook,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "langgraph-checkpoint";
+  version = "1.0.3";
+  pyproject = true;
+
+  disabled = pythonOlder "3.10";
```

```suggestion
  disabled = pythonOlder "3.9";
```
https://github.com/langchain-ai/langgraph/blob/855139d869aa0a7391b73f0708c9d1168e2355f4/libs/checkpoint/pyproject.toml#L12

---

Fixed.